src/ethereum/frontier/spec.py

```diff
@@ -90,38 +89,6 @@ def apply_fork(old: BlockChain) -> BlockChain:
     new : `BlockChain`
         Upgraded block chain object for this hard fork.
     """
-    genesis = genesis_configuration("mainnet.json")
```

Comment:
Leaving this comment here since I cannot comment on unchanged lines.

With the new changes, we no longer need L891, L892 in the frontier code do we?

Reply:
I've addressed this independently in #599.